src/epics/login/silentLogin.epic.ts

```diff
@@ -27,17 +27,17 @@ export const silentLogin = (action$: AnyAction, state$: any) => action$.pipe(
         .pipe(
             switchMap(() => userService.getUser()
                 .pipe(
-                    map(authActions.loginSuccess),
                     tap(() => console.log('Silent login success using authToken')),
+                    mergeMap((data) => [authActions.loginSuccess(data), authActions.refreshAuthStart(), userActions.getProjectList()]),
                     catchError((err: any) => {
                             console.log('Unable to login using authToken: ', err.message);
                             return authService.refreshAuth()
                                 .pipe(
                                     tap((e) => console.log(e)),
                                     switchMap(() => userService.getUser()
                                         .pipe(
-                                            mergeMap((data) => [authActions.loginSuccess(data), userActions.getProjectList()]),
                                             tap(() => console.log('Silent login success using refreshToken')),
+                                            mergeMap((data) => [authActions.loginSuccess(data), authActions.refreshAuthStart(), userActions.getProjectList()]),
```

Comment:
Same here, I wouldn't trigger this action manually but simply let the epic do the work